### DISCUSSION TITLE
[mini.files] Are your icons small as well?

### DISCUSSION THREAD

**Discussion by pkazmier** [nvim-mini/mini.nvim#1055] [Q&A] [answered]
For those that use `mini.files`, are your icons using the single width variant instead of the double width variant? I looked in the source code and there is a space appended when the icon is selected, so I'm not sure why my terminal (wezterm) is not rendering the double width icon. As you can see in the screenshot, it **does** render the double width icon in the tabline, so I'm not really sure what is going on here.
<img width="1017" alt="image" src="https://github.com/user-attachments/assets/0eaaf2c7-a0cf-487c-9d6a-09a58e6a3621">

And then I fired up Kitty to see what happens there. It does render the double width icons in `mini.files`, but `mini.tabline` oddly renders the **active** tab with the single width icon. As I switch between buffers, the active buffer's icons shrinks. Here is a video:
https://github.com/user-attachments/assets/17c11b19-921a-45ca-adc4-29d9df8dcbd4

**Comment by echasnovski** [comment, marked as answer]
Does setting 'winblend' to 0 in 'mini.files' floating window help?

**Reply by pkazmier** [reply]
Yes, it does fix it!

**Comment by pkazmier** [comment]
I never noticed that icons were also small in my `cmp` popup menus. I think I was too enamored with the faux transparency to notice. All of this stems from me enabling `extra_ui` in `mini.basics`. I've disabled while I determine what I like better: big icons or faux transparency? 😄

**Reply by echasnovski** [reply]
Yeah, so have I :) Not sure about the future of it as part of `extra_ui` because of that.
